api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
                     - authorizationCode
+                    - clientId
                 - type: object
                   properties:
                     clientId:
                       type: string
+                      $ref: '#/components/schemas/oauth2ClientId'
                     clientSecret:
                       type: string
```

Comment:
whats the diff between INVALID_REFRESH_TOKEN_ERROR and UNKNOWN_REFRESH_TOKEN_ERROR?

Reply:
- `UNKNOWN_REFRESH_TOKEN_ERROR ` notifies that the refresh_token is not found. 
-  `INVALID_REFRESH_TOKEN_ERROR`  is to notify that the refresh_token is expired.